package.json

```diff
@@ -4,7 +4,8 @@
   "description": "TRIM API wrapper",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "./node_modules/mocha/bin/mocha specs.js",
+    "test:watch": "./node_modules/nodemon/bin/nodemon.js --exec \"npm run test\" specs.js"
```

Comment:
Yeah, Richie and I did quite a bit of iteration on test style in that project. Do you remember which tests were the better ones for Ersin to look at in there @digitalsadhu? From memory there are some older ones that we were... less proud of.